chcemvediet/locale/sk/LC_MESSAGES/django.po

```diff
@@ -8,7 +8,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: PACKAGE VERSION\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2016-02-21 18:51+0100\n"
+"POT-Creation-Date: 2020-05-18 17:57+0200\n"
```

Comment:
@martinmacko47 Jedina zmena v `.po` suboroch v priecinku `/poleno/` bola novo vygenerovana hodnota `POT-Creation-Date`. V `/poleno/` priecinku som teda neaktualizoval datum lebo sa nepridali ziadne nove preklady ani ziadne ine upravy. Alebo mam aktualizovat aj ten datum?

Reply:
@viliambalaz To je dobre. Ak sa v `.po` subore zmeni iba datum, tak zbytocne ich aktualizovat.